### PR TITLE
responsibilities for job

### PR DESCRIPTION
**Pull request by ciarasugrue**

### CODE REVIEW

**Review by dianemorrison — APPROVED** [review]

File: controller/JobController.ts [hunk 1]
```diff
@@ -14,10 +14,31 @@ module.exports = function(app: Application){
...
     }
     
   });
+
+  app.get('/job-spec/:jobID', roleAccess([UserRole.Admin,UserRole.User]), async (req: Request, res: Response) => {
+    const jobID = parseInt(req.params.jobID);
+
+    try {
+      const job = await jobservice.getJobResponsibilities(jobID); 
+      const responsibilities = await jobservice.getJobResponsibilities(jobID);
```
Comment (shaunganley) [inline controller/JobController.ts:31]: these both call the same thing, you shouldn't need to have 2 calls here

File: service/ResponsibilityService.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import { Responsibility } from '../model/Responsibility';
+import axios from 'axios';
+axios.defaults.baseURL = process.env.API_URL;
+
+
+export default class ResponsibilityService {
+
+  URL: string = '/job-spec'; 
+
+  async getResponsibilitiesForJob(): Promise<Responsibility[]>{
```
Comment (shaunganley) [inline service/ResponsibilityService.ts:12]: this class isn't being called, think you can delete

**Review by shaunganley — APPROVED** [review]
remove needed code

File: controller/JobController.ts (hunk 1 above)
Comment (ciarasugrue) [reply controller/JobController.ts:31]: removed line 30

File: service/ResponsibilityService.ts (hunk 2 above)
Comment (ciarasugrue) [reply service/ResponsibilityService.ts:12]: Class and corresponding unit tests deleted

**Pull request merged**